ocaml/xapi/message_forwarding.ml

```diff
@@ -5287,6 +5287,20 @@ functor
         info "SDN_controller.forget: sdn_controller = '%s'"
           (sdn_controller_uuid ~__context self) ;
         Local.SDN_controller.forget ~__context ~self
+
+      let open_port ~__context ~self ~protocol ~port =
+        info "SDN_controller.open_port: sdn_controller = '%s', protocol = '%s', port = '%s'"
+          (sdn_controller_uuid ~__context self)
+          (Record_util.sdn_port_protocol_to_string protocol)
+          (Int64.to_string port);
+        Local.SDN_controller.open_port ~__context ~self ~protocol ~port
```

Comment:
This would open the port on only one host (the one receiving the API call, which may be a slave).
Do you need this port opened everywhere?

Reply:
Yes I noticed it while I was testing. I need to do it on all the pools host indeed but I think I know how since I dit it [here](https://github.com/xapi-project/xen-api/pull/4164/files#diff-9a639b670fb7e2432894cc91df8b27fbR3370-R3371)